packages/language-server/test/completion-items-classes-spec.ts

```diff
@@ -549,4 +549,92 @@ describe("the UI5 language assistant Code Completion Services - classes", () =>
       ],
     });
   });
+
+  it("will not return deprecated suggestions according to settings", async () => {
```

Comment:
I think these tests belong in xml-view-completion
Note we are testing the completions identity/data not the LSP related transformed data.

Reply:
Moved (though I kept a couple of smoke tests to make sure we actually pass the settings)